api/src/test/java/org/openmrs/OrderGroupTest.java

```diff
@@ -0,0 +1,39 @@
+package org.openmrs;
+
+import java.util.Date;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+import org.openmrs.api.builder.OrderBuilder;
+import org.openmrs.test.BaseContextSensitiveTest;
+
+/**
+ *
+ * @author User
+ */
+public class OrderGroupTest extends BaseContextSensitiveTest{
```

Comment:
I don't think test requires any access to the DB, so it should not extend `BaseContextSensitiveTest`